sbi/inference/snpe/snpe_c.py

```diff
@@ -83,53 +83,53 @@ def _get_log_prob_proposal_posterior(self, inputs, context, masks):
         estimator.
 
         Args:
-            inputs: torch.Tensor Batch of parameters.
-            context: torch.Tensor Batch of observations.
+            theta: torch.Tensor Batch of parameters theta.
```

Comment:
No type needed (rather in the signature).

Reply:
This should be without type and with lower case `batch`, is this fixed? I am always not sure whether github shows the most recent version?